openhcl/virt_mshv_vtl/src/lib.rs

```diff
@@ -1148,6 +1148,8 @@ pub struct UhPartitionNewParams<'a> {
     pub use_mmio_hypercalls: bool,
     /// Intercept guest debug exceptions to support gdbstub.
     pub intercept_debug_exceptions: bool,
+    /// Allocator for DMA pages to be preserved during servicing.
+    pub dma_pages_pool: Option<fixed_pool_alloc::FixedPoolAllocator>,
```

Comment:
Why is this here?

Reply:
Yes, please remove.

Reply:
Looks like a leftover before replacing allocator with allocator spawner. Will remove.

Reply:
Removed